apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,57 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel = "stylesheet" type = "text/css" href = "./css/main.css" />
 </head>
 <body>
```

Comment:
You are missing a basic structure of the site using the ```header``` and ```main``` tags:
```
<header></header>
<main></main>
```

And inside will go everything else. Usually you will have a ```footer``` as well.